### DISCUSSION TITLE
iOS 14 Public Beta 6 Time Picker

### DISCUSSION THREAD

**Discussion by HenryLeC** [xamarin/Xamarin.Forms#11966] [General] [answered]
The time picker in iOS Public Beta 6 does not work in Xamarin.Forms, This is very important as the new version of Xcode 12 only allows testing on iOS 14 and there is no way to roll back to Public Beta 5

![2020-08-30_10-58-35-AM](https://user-images.githubusercontent.com/32208783/91666592-02d59080-eacc-11ea-99a0-3da97305517c.png)
![2020-08-30_10-58-40-AM](https://user-images.githubusercontent.com/32208783/91666595-06691780-eacc-11ea-9250-3a97d4120439.png)

**Comment by davidortinau** [comment, marked as answer]
Thanks @HenryLeC  for this and filing the issue #11963. I added the native iOS code that controls the style for example there as well.
